docs/guides.md

```diff
@@ -78,27 +78,58 @@ Before we jump into writing some exchanges, there are a couple of
 patterns and limitations that always remain the same when writing
 an exchange.
 
+### Forward Operation Streams. Return OperationResult Streams.
+
 For reference, this is a basic template for an exchange:
 
 ```js
 const noopExchange = ({ client, forward }) => {
-  return operations$ => {
-    return forward(operations$);
+  return operation$ => { // <-- The ExchangeIO function
```

Comment:
I think these comments are actually not indicating a lot of information if we’re repeating them too often, so let’s maybe annotate “Exchange” and “ExchangeIO” separately

Reply:
Agreed that they aren't adding information.  I think annotating "Exchange" is unnecessary since exchanges are called fooExchange by convention.

"ExchangeIO" annotations repeat a few times (not the whole doc) for three reasons:
- it is always anonymous by convention
- learners are learning enough new concepts in this section for their working memory to drop the first instances before retaining it
- Repetition facilitates memory retention

Reply:
Fair, that’s a great point 👍 let’s keep it!